src/handlers/local-upload.ts

```diff
@@ -68,6 +77,7 @@ async function pollForAssetReady(filePath: string, asset: Asset) {
       externalIds: {
         playbackId,
       },
+      blurDataURL,
```

Comment:
Feels a little weird to put this at the top level? Should we consider making `externalIds` something like `uploaderMetadata` or something like that and putting this there too?

Reply:
I agree, that sounds good to me